src/js/components/deployments/__snapshots__/deploymentitem.test.js.snap

```diff
@@ -7,7 +7,10 @@ exports[`DeploymentItem Component renders correctly 1`] = `
   >
     Release
   </span>
-  <div>
+  <div
+    class="text-overflow"
+    style="white-space: initial;"
```

Comment:
It would also work without the "white-space: initial;" - that would leave it as a single line with truncated text. Might be preferable if there are many rows of deployments? What do you think?
![Screenshot 2022-06-16 at 11 41 55](https://user-images.githubusercontent.com/15045782/174042417-e14d4b0b-a985-478f-a74e-a82f884f7d1f.png)